lib/src/subscription.dart

```diff
@@ -96,7 +94,7 @@ class SubscriptionImpl implements Subscription {
     }
     _state = _SubscriptionState.unsubscribed;
 
-    if (!_client.connected) {
+    if (!_client.connected!) {
```

Comment:
Since the `connected` property can't be null, '!' is not needed here.

Reply:
Fixed, thanks!